instrumentation/http-tests/src/main/java/brave/test/http/ITServlet25Container.java

```diff
@@ -71,9 +77,13 @@
   }
 
   static class ExceptionServlet extends HttpServlet {
-    @Override
-    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException {
-      throw new IOException(); // null exception message!
+    @Override protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+      throws UnavailableException {
+      // Change the status from 500 to 503
+      req.setAttribute(RequestDispatcher.ERROR_STATUS_CODE, 503);
+      // TODO: org.eclipse.jetty.server.HttpChannelState.onError() clobbers ^^ and hard-codes
```

Comment:
this was a lot of time lost trying to figure out where it clobbers the code, only in sync errors, not async ones! :P